src/interfaces/podcast.ts

```diff
@@ -6,5 +6,6 @@ export interface Podcast {
     slug?: string;
     thumbnailUrl: string;
     title: string;
-    url: string;
+    url?: string;
+    podcastId: string;
```

Comment:
what is podcastId ?

Reply:
Oh I forgot I put that there. It is the oid from mongodb. After seeing the duplicate key errors in the front-end, I put this there to resolve any potential duplicate key errors when we render podcasts card components. However, since you mentioned that all podcasts are unique, then having the keys as mediaType + title should be fine. I shall take it out because it is not being used

Reply:
Yeah also seems like for Twitch it is not yet unique to where we can't just use the slug or something